### ISSUE TITLE
[Question] Low performance on specific picture format (portrait vs. landscape)

### ISSUE THREAD

**Issue by valentindbdg** [AlexeyAB/darknet#8313]
Hello,

I am training a yolo-v4-tiny model on a custom dataset:

- dataset: validation set of COCO 2017, divided into train, validation and test sets
- one class: "person"
- around 3000 images

Parameters used:
- 320*320
- max_batch = 6000
- step1 = 0.8 * max_batch
- step2 = 0.9 * max_batch
- num_classes = len(labels)
- num_filters = (num_classes + 5) * 3
- batch = 64
- subdivisions = 16

I loaded my predictions into Fiftyone to visualize my predictions. 
I noticed a difference of performance between pictures that are in landscape and portrait formats.

Landscape format:

![good detection_scenary_2](https://user-images.githubusercontent.com/33783419/147669258-cf81a707-fa5b-4e56-886d-975938129ad0.png)
![good detection_scenary_3](https://user-images.githubusercontent.com/33783419/147669262-1f367253-4da4-4ca7-b120-9ccf2e1289fc.png)
![good detection_scenary_1](https://user-images.githubusercontent.com/33783419/147669266-a6cf75b1-dd87-4272-9829-00ad89cec6ff.png)

Portrait format:

![bad detection_portrait_5](https://user-images.githubusercontent.com/33783419/147670654-dedac1bc-4d33-42d2-9b87-bb429952354a.png)
![bad detection_portrait_6](https://user-images.githubusercontent.com/33783419/147670658-46548a16-79dc-4958-87a0-0bfc1c38613d.png)
![bad detection_portrait_1](https://user-images.githubusercontent.com/33783419/147670660-657c924d-db0d-4d34-a924-52fbea4607af.png)
![bad detection_portrait_2](https://user-images.githubusercontent.com/33783419/147670664-940d879d-72e1-4953-9295-8afa9e82589e.png)
![bad detection_portrait_3](https://user-images.githubusercontent.com/33783419/147670666-4393dd57-5c9b-467b-8301-a025d1297ce9.png)
![bad detection_portrait_4](https://user-images.githubusercontent.com/33783419/147670668-e4ac0118-202f-42a9-9b80-8e8cdb04bb61.png)

Does it make sens to think that this difference of performance is due to an unbalance dataset (number of portrait vs. landscape format images) and that adding images in the training set with portrait format would improve the model's perfomances? Or am I missing anything?

My goal is to improve performances by choosing good data instead of using big data.

Thank you for your help!
